http/headers/Save-Data.json

```diff
@@ -14,11 +14,11 @@
               "version_added": "≤79"
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
```

Comment:
Can you provide a source for this change?

Reply:
One way to test this is to open a page which pings back your headers.

Testing steps for Android devices:
1. On your Android device, open settings and find "Data saver", enable it
2. Disable Wi-Fi and enable mobile data
3. Open the browser and go to https://www.whatismybrowser.com/detect/what-http-headers-is-my-browser-sending
4. Observe `SAVE-DATA` header with value `on`

Testing steps for Opera:
1. Open Android device, and open Opera settings (not System settings). enable "Data saver"/"Turbo" mode
2. You may use Wi-Fi or mobile data, this does not matter
3. Open the browser and go to https://www.whatismybrowser.com/detect/what-http-headers-is-my-browser-sending
4. Observe `SAVE-DATA` header with value `on`

Testing results:
Chrome 108.0.5357.47 - supports
Opera 72.3.3767.68685 - supports
Firefox 107.1.0 - does not support
Brave 1.45.127 (Chromium 107.0.5304.110) - does not support

More info:

`Save-Data` was pioneered by Chromium as part of Google Chrome's for Android "Data Saver" mode which later became "Lite" mode.[1][2] `Save-Data` header also migrated into Opera as part of "Opera Turbo" on and Yandex as part of "Turbo" mode on both desktop and mobile.[2] These features also included a proxy which would compress the data, and reduce image quality. I could not find no reference to Firefox ever implementing this header.

Please note that Chrome 100 had removed the "Lite" mode and associated UI,[3] but the `Save-Data` header survived and is now based on Android Data Saver API.[4] This is just a simple preference now, without any proxies in Chrome. Opera still retained the UI and proxy functionality.

[1] https://css-tricks.com/help-users-save-data/
[2] https://web.dev/optimizing-content-efficiency-save-data
[3] https://support.google.com/chrome/thread/151853370
[4] https://chromium-review.googlesource.com/c/chromium/src/+/3573287